### PR TITLE
Create the new "Permissions and Privacy" screen

### PR DESCRIPTION
**Pull request by anikiki**
Task/Issue URL: https://app.asana.com/0/0/1204667877992130/f 

### Description
Created the new "Permissions and Privacy" screen.

### Steps to test this PR

- [ ] Install from  this branch and go to the `Settings` screen. 
- [ ] Tap on the `Permissions and Privacy` item and notice the new screen.
- [ ] Tap on each item on the `Permissions and Privacy` screen and notice they behave as before.

### UI changes
| New screen  | New screen scrolled to bottom |
| ------ | ----- |
|permissions_and_privacy_1|permissions_and_privacy_2|

### CODE REVIEW

**Comment by anikiki** [comment]
Current dependencies on/for this PR:
* develop
  * **PR #3226** <a href="https://app.graphite.dev/github/pr/duckduckgo/Android/3226" target="_blank">Graphite</a> 
    * **PR #3230** <a href="https://app.graphite.dev/github/pr/duckduckgo/Android/3230" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/duckduckgo/Android/3230?utm_source=stack-comment).

File: app/src/main/java/com/duckduckgo/app/permissionsandprivacy/PermissionsAndPrivacyActivity.kt [hunk 1]
```diff
@@ -0,0 +1,348 @@
...
+import com.duckduckgo.di.scopes.ActivityScope
+import com.duckduckgo.mobile.android.ui.view.dialog.RadioListAlertDialogBuilder
+import com.duckduckgo.mobile.android.ui.viewbinding.viewBinding
+import javax.inject.Inject
+import kotlinx.coroutines.flow.launchIn
+import kotlinx.coroutines.flow.onEach
+import timber.log.Timber
+
+@InjectWith(ActivityScope::class)
+class PermissionsAndPrivacyActivity : DuckDuckGoActivity() {
```
Comment (anikiki) [inline app/src/main/java/com/duckduckgo/app/permissionsandprivacy/PermissionsAndPrivacyActivity.kt:61]: Moved from `SettingsActivity`.

File: app/src/main/java/com/duckduckgo/app/permissionsandprivacy/PermissionsAndPrivacyViewModel.kt [hunk 2]
```diff
@@ -0,0 +1,281 @@
...
+import timber.log.Timber
+
+@ContributesViewModel(ActivityScope::class)
+class PermissionsAndPrivacyViewModel @Inject constructor(
+    private val settingsDataStore: SettingsDataStore,
+    private val pixel: Pixel,
+    private val gpc: Gpc,
+    private val featureToggle: FeatureToggle,
+    private val autoconsent: Autoconsent,
+) : ViewModel() {
```
Comment (anikiki) [inline app/src/main/java/com/duckduckgo/app/permissionsandprivacy/PermissionsAndPrivacyViewModel.kt:52]: Moved from `SettingsViewModel`.

File: app/src/test/java/com/duckduckgo/app/permissionsandprivacy/PermissionsAndPrivacyViewModelTest.kt [hunk 3]
```diff
@@ -0,0 +1,380 @@
...
+import org.mockito.Mock
+import org.mockito.MockitoAnnotations
+import org.mockito.kotlin.any
+import org.mockito.kotlin.eq
+import org.mockito.kotlin.never
+import org.mockito.kotlin.verify
+import org.mockito.kotlin.whenever
+
+@OptIn(ExperimentalCoroutinesApi::class)
+class PermissionsAndPrivacyViewModelTest {
```
Comment (anikiki) [inline app/src/test/java/com/duckduckgo/app/permissionsandprivacy/PermissionsAndPrivacyViewModelTest.kt:51]: Most tests moved from `SettingsViewModelTest`. Added a few tests that were missing (no new functionality).

**Review by malmstein — APPROVED** [review]
Looking good!

**Pull request merged**